runtime/src/lib.rs

```diff
@@ -127,13 +127,13 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("dock-testnet"),
     impl_name: create_runtime_str!("dock-testnet"),
     authoring_version: 1,
-    spec_version: 1,
+    spec_version: 2,
     impl_version: 1,
     transaction_version: 1,
     apis: RUNTIME_API_VERSIONS,
 };
 
-const MILLISECS_PER_BLOCK: u64 = 6000;
+const MILLISECS_PER_BLOCK: u64 = 3000;
```

Comment:
I fail to see how this is related to a version upgrade. Is it just to imitate the newest node template?

Reply:
Decreasing block time as we see delays in SDK integration tests. The delay will also be seen by testnet users so changing it for now. In a bigger network, this might cause more uncles to be created but should be fine for now.